types/smooth-scroll/test/smooth-scroll.test.ts

```diff
@@ -0,0 +1,36 @@
+import * as SmoothScroll from "smooth-scroll";
```

Comment:
Since this is a callable or constructable entity, it isn't legal in ES2015 to import and call/construct this with a namespace import like `import * as foo from 'foo'`, so could you please convert this to either
```javascript
import foo = require('foo')
```
or the following with esModuleInterop turned on
```javascript
import foo from 'foo';
```